repo_resource/check.py

```diff
@@ -33,13 +35,23 @@ def check(instream) -> list:
 
     config = common.source_config_from_payload(payload)
 
+    standard_versions = []
+    for v in payload.get('versions', []):
+        standard_versions.append(common.Version(v['version']).standard())
+
     if config.private_key != '_invalid':
         common.add_private_key_to_agent(config.private_key)
 
+    jobs = config.jobs
+    check_jobs = config.check_jobs or jobs*2 or DEFAULT_CHECK_JOBS
+
     try:
-        repo = common.Repo()
-        repo.init(config.url, config.revision, config.name, config.depth)
-        repo.sync(jobs=config.jobs)
+        repo = common.Repo(config.url,
```

Comment:
Why the formatting change?

Doing arbitrary format changes hurts the review as it makes the patch bigger (and thus more difficult to review)

If we want to keep the formatting change, should do it in a separate commit.

https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/

Reply:
More arguments so the line was too long to be PEP8 compliant